SDKv3_Examples/app/src/main/java/com/mappedin/sdkv3_examples/MainActivity.kt

```diff
@@ -85,6 +97,9 @@ class MainActivity : AppCompatActivity() {
                     supportActionBar?.title = map.name
                 }
                 println("MPIMap Changed: " + Json.encodeToString(map))
+
+                // Create an MPICoordinate from Latitude and Longitude
+                val coord = map.createCoordinate(43.5214,-80.5369)
```

Comment:
Is there anything we can use the coordinate that's visible? I forgot the point of MPICoordinate

Reply:
Don't think so, I added variables for x and y values but that's all I can think of